includes/components/class-user.php

```diff
@@ -342,7 +342,6 @@ public function update_profile_fields( $user_id ) {
 
 		// Delete verification key.
 		if ( hp\get_array_value( $_POST, 'hp_email_verified' ) && get_user_meta( $user_id, 'hp_email_verify_key', true ) ) {
-			delete_user_meta( $user_id, 'hp_email_verified' );
```

Comment:
That's ok, this was required to delete the unnecessary meta saved by a new meta box API (it runs after the meta box update callback and clears meta since after activation it's not used anywhere).